praw/models/reddit/subreddit.py

```diff
@@ -3,13 +3,14 @@
 from copy import deepcopy
 from json import dumps, loads
 from os.path import basename, dirname, join
+import socket
```

Comment:
For consistency `import ...` should occur before `from ... import ...` statements. I think I used to do it the other way around, but it seems like this is the more agreed upon order.

Reply:
OK, thanks. I was alphabetizing on package name. I'll change that.